gpkit/interactive/plot_sweep1d.py

```diff
@@ -0,0 +1,48 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+import numpy as np
+from ..exceptions import InvalidGPConstraint
+from ..tools import autosweep_1d
+
```

Comment:
`sol.plot_sweep(xvar, yvar=None)` would plot xvar vs yvar and use cost as the default yvar.